modules/apps/object/object-test/src/testFunctional/paths/VisualizeObjectDisplayPage.path

```diff
@@ -17,7 +17,7 @@
 	</tr>
 	<tr>
 		<td>SELECT_OBJECT</td>
-		<td>//span[contains(text(),'${key_pluralLabelName}')]</td>
+		<td>//div[contains(@class,'collapse menubar-collapse')]//a[contains(.,'${key_pluralLabelName}')]</td>
```

Comment:
Do we need to include `collapse` the path? Can we just use:
`//div[contains(@class,'menubar-collapse')]//a[contains(.,'${key_pluralLabelName}')]`